packages/mask/src/social-network/defaults/inject/PageInspector.tsx

```diff
@@ -20,8 +23,10 @@ export function injectPageInspectorDefault<T extends string>(
             .appendChild(document.createElement('div'))
             .attachShadow({ mode: Flags.using_ShadowDOM_attach_mode })
 
-        createReactRootShadowed(dom, { signal, key: 'page-inspector' }).render(<PageInspectorDefault />)
+        createReactRootShadowed(dom, { signal, key: 'page-inspector' }).render(
+            <HashRouter>
```

Comment:
Wait, this does not look right. Why you're adding `HashRouter` in the SNS Adaptor? That's definitely wrong.

Reply:
@guanbinrui

Reply:
still not resolved 👀